utils/errors/notifier/notifier.go

```diff
@@ -109,21 +117,27 @@ func convToSentry(in errors.ErrorExt) *raven.Stacktrace {
 	return out
 }
 
-func parseRawData(ctx context.Context, rawData ...interface{}) map[string]interface{} {
+func parseRawData(ctx context.Context, rawData ...interface{}) (map[string]interface{}, map[string]string) {
 	m := make(map[string]interface{})
+	t := make(map[string]string)
 	for pos := range rawData {
 		data := rawData[pos]
 		if _, ok := data.(context.Context); ok {
 			continue
 		}
+		if sentryTags, ok := data.(Tags); ok {
+			for k, v := range sentryTags {
```

Comment:
Will it be possible for us to do this with interfaces ?

Reply:
Do you mean define an interface such as `isTags`
Any struct that implements this interface can be validated and accepted as tags here?

Reply:
Or do you mean assign tags by type assertion without the for loop?

Reply:
First one, something like isTags

Reply:
added isTags for generic tags abstraction.
and a more specific isSentryTags for sentry.